pkg/operator/ceph/config/monstore.go

```diff
@@ -93,7 +94,11 @@ func (m *MonStore) SetIfChanged(who, option, value string) (bool, error) {
 // Set sets a config in the centralized mon configuration database.
 // https://docs.ceph.com/docs/master/rados/configuration/ceph-conf/#monitor-configuration-database
 func (m *MonStore) Set(who, option, value string) error {
-	logger.Infof("setting option %q (user %q) to the mon configuration database", option, who)
+	if logger.LevelAt(capnslog.TRACE) {
+		logger.Tracef("setting option %q (user %q) = %q to the mon configuration database", option, who, value)
```

Comment:
How about the value before the parenthesis in all these statements?
```suggestion
		logger.Tracef("setting option %q = %q (user %q) to the mon configuration database", option, value, who)
```

Reply:
done